doc/styleguide.md

```diff
@@ -0,0 +1,248 @@
+# Node.js Documentation Style Guide
+
+These are the guidelines for writing Node.js documentation.
+
+## Headings
+
+* Each page must have a single `#`-level title at the top.
+* Chapters in the same page must have `##`-level headings.
+* Sub-chapters need to increase the number of `#` in the heading according to
+  their nesting depth.
+* The page's title must follow [APA title case][title-case].
+* All chapters must follow [APA sentence case][sentence-case].
+
+Using a hypothetical `Quick Start` as an example:
+
+```markdown
+# How to write a test for the Node.js project
+
+...
+
+## What is a test?
+
+...
+
+## Test directory structure
+
+...
+
+## Test structure
+
+...
+
+### **Lines 1-3**
+
+...
+
+### **Lines 5-6**
+
+...
+```
+
+For API references, there are exceptions to this rule.
+
+## Markdown rules
+
+This repository uses the [`remark-preset-lint-node`][remark-preset-lint-node]
+package to enforce consistent Markdown styling. For the exact rules, see
+[nodejs/remark-preset-lint-node][nodejs/remark-preset-lint-node]. Additionally,
+we're beginning to use [docs-parser][docs-parser] to parse the Markdown
+into an API.
+
+There are a few style guidelines that aren't covered by the linter rules:
+
+* Use `sh` instead of `cmd` in code blocks (due to the syntax highlighter).
+* Keep line lengths between 80 and 100 characters if possible for readability
+  purposes.
+* No nesting lists more than 2 levels (due to the markdown renderer).
+* All `js` and `javascript` code blocks are linted with
+  [standard-markdown](https://www.npmjs.com/package/standard-markdown).
+* For unordered lists, use asterisks instead of dashes.
+
+## Picking words
+
+* Use "will" over "would" when describing outcomes.
+
+## API references
+
+The following rules only apply to the documentation of APIs.
+
+### Title and description
```

Comment:
```suggestion
### Title and Description
```

APA title case would need to be applied to all of these titles.

How can we lint for this? I would like to do so.

Reply:
> APA title case would need to be applied to all of these titles.

This is not the title of the page and per the text above, should be sentence case.

Reply:
You are correct in that this not being the title of the page; however, it is the title of a section of the page. We can call them &ldquo;chapters&rdquo; as @bnb proposes.

Other words for this could be &ldquo;segments&rdquo;, &ldquo;sections&rdquo;, etc&hellip;